spiffetls/tests/resources/ca.crt

```diff
@@ -0,0 +1,19 @@
+-----BEGIN CERTIFICATE-----
+MIIDATCCAemgAwIBAgIUAUQ8q+K5VmiFEfHTwxrfhweVIQowDQYJKoZIhvcNAQEL
+BQAwEDEOMAwGA1UEAwwFbXktY2EwHhcNMjEwMzE4MTI1NTMxWhcNMjIwMzE4MTI1
+NTMxWjAQMQ4wDAYDVQQDDAVteS1jYTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCC
+AQoCggEBALvAH5/6V4hUrjLJvlITD+h0KD5l7COsmYXfD1AwdqymojxARQ2vokNT
+sVBB8YGmvHuRNS1MtH2AkCr8iMsl03HUigGfsw0PGH/Ldaf8SenSIgvY4V3EaTKU
+FCsnm6A/z3v33AmaeiE7Wzn7t0PeptlRiyO5rwfd9zTGd9eTuY5DyiZaP7qGnBfY
+Dr8Zb/COCTRiUtFy2+1BnuQUW1WCBfuywKa4y504qvbz3sZz9aO+n9FSlKBjoBHJ
+yJCDSaavb1Tmtgqb47iwgaI34xoPwt0RTlpKEIMA67+GxYTrABC0fU9I9mVoF/e3
+LKtfRKytazuEtzPCfNrSjWB1y/nBP1ECAwEAAaNTMFEwHQYDVR0OBBYEFAo49sQQ
+gwtsSch6K0zGRoX6N/DeMB8GA1UdIwQYMBaAFAo49sQQgwtsSch6K0zGRoX6N/De
+MA8GA1UdEwEB/wQFMAMBAf8wDQYJKoZIhvcNAQELBQADggEBACDFZzQe1Rpv5MSL
+CXfxdNtxbfqO1i0t+nCNy7N6zIaRgH5+Rj7/oNE0c+kF3n1DucCjm58G4Gss4CVh
+A07s6CayYw4JvlU2HNI9HUowd77eXBg1ZIpdforn9YXHgsUMBybOffouv8zuTR7u
+GeE5ufhipBcBRE+1HhIspAkAnx+f2qvBsn65vEagAPKqIw/BlP9ISz8aIGub/d4X
+V3MqqEMJ9BPhBoXSAjoAOq71WjPsRh1P3p2dRIpf3Qz6qMkvMN7uX6xyceKJnhoF
+1Vi/2nbDTDGGmCh83MdmnQmyDVSiKouKpRqXSm2GUUGAG0620yDRcXZZ2xY9wEfl
+ultQtEs=
+-----END CERTIFICATE-----
```

Comment:
No newline at end, doesn't this cause some issues? tests run fine, so I might be misremembering